api/turing/api/appcontext.go

```diff
@@ -44,7 +44,7 @@ func NewAppContext(
 	db *gorm.DB,
 	cfg *config.Config,
 	authorizer *middleware.Authorizer,
-	vaultClient vault.VaultClient,
+	vaultClient vault.Client,
```

Comment:
This change is made due to an update in the gojek/mlp version which feature new struct names in the vault package.